[2025-01-05T21:17:07.978+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-01-05T21:17:07.990+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: raw_banking_tinkoff.file_processing manual__2025-01-05T21:17:06.492084+00:00 [queued]>
[2025-01-05T21:17:07.994+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: raw_banking_tinkoff.file_processing manual__2025-01-05T21:17:06.492084+00:00 [queued]>
[2025-01-05T21:17:07.994+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-01-05T21:17:07.998+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): file_processing> on 2025-01-05 21:17:06.492084+00:00
[2025-01-05T21:17:08.002+0000] {logging_mixin.py:188} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=193) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-01-05T21:17:08.003+0000] {standard_task_runner.py:63} INFO - Started process 195 to run task
[2025-01-05T21:17:08.002+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'raw_banking_tinkoff', 'file_processing', 'manual__2025-01-05T21:17:06.492084+00:00', '--job-id', '64', '--raw', '--subdir', 'DAGS_FOLDER/raw_banking_tinkoff.py', '--cfg-path', '/tmp/tmpmdn9y9lg']
[2025-01-05T21:17:08.003+0000] {standard_task_runner.py:91} INFO - Job 64: Subtask file_processing
[2025-01-05T21:17:08.022+0000] {task_command.py:426} INFO - Running <TaskInstance: raw_banking_tinkoff.file_processing manual__2025-01-05T21:17:06.492084+00:00 [running]> on host 0a82d8fba642
[2025-01-05T21:17:08.054+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='nmashtakov' AIRFLOW_CTX_DAG_ID='raw_banking_tinkoff' AIRFLOW_CTX_TASK_ID='file_processing' AIRFLOW_CTX_EXECUTION_DATE='2025-01-05T21:17:06.492084+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-05T21:17:06.492084+00:00'
[2025-01-05T21:17:08.054+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-01-05T21:17:08.159+0000] {python.py:237} INFO - Done. Returned value was:            Дата операции  ...              load_dt
0    31.12.2024 17:39:21  ...  05.01.2025 21:17:08
1    31.12.2024 17:38:42  ...  05.01.2025 21:17:08
2    31.12.2024 17:37:14  ...  05.01.2025 21:17:08
3    31.12.2024 17:28:45  ...  05.01.2025 21:17:08
4    30.12.2024 20:43:40  ...  05.01.2025 21:17:08
..                   ...  ...                  ...
104  01.12.2024 19:43:31  ...  05.01.2025 21:17:08
105  01.12.2024 12:09:00  ...  05.01.2025 21:17:08
106  01.12.2024 12:03:44  ...  05.01.2025 21:17:08
107  01.12.2024 11:44:41  ...  05.01.2025 21:17:08
108  01.12.2024 11:44:26  ...  05.01.2025 21:17:08

[109 rows x 16 columns]
[2025-01-05T21:17:08.167+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-01-05T21:17:08.195+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=raw_banking_tinkoff, task_id=file_processing, run_id=manual__2025-01-05T21:17:06.492084+00:00, execution_date=20250105T211706, start_date=20250105T211707, end_date=20250105T211708
[2025-01-05T21:17:08.237+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-01-05T21:17:08.247+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-01-05T21:17:08.247+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
